fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -24,13 +24,13 @@ var getCookie = function (name) {
 var parseCookies = function () {
 	var cookieData = (typeof document.cookie === 'string' ? document.cookie : '').trim();
 
-	return (cookieData ? cookieData.split(';') : []).reduce(function (cookies, cookieString) {
-		var cookiePair = cookieString.split('=');
+      return (cookieData ? cookieData.split(';') : []).reduce(function (cookies, cookieString) {
+              var cookiePair = cookieString.split('=');
 
-		cookies[cookiePair[0].trim()] = cookiePair.length > 1 ? cookiePair[1].trim() : '';
+              cookies[cookiePair[0].trim()] = cookiePair.length > 1 ? cookiePair[1].trim() : '';
 
-		return cookies;
-	}, {});
+              return cookies;
+      }, {});
 };
```

Comment:
Deze regels zijn niet aangepast, maar in de files changed wordt dit wel aangegeven.